### PR TITLE
API: We now support two more timeline api endpoints

### PR DESCRIPTION
**Pull request by annando**
We now support the `timelines/home` and `timelines/tag/{hashtag}` endpoints.

### CODE REVIEW

**Comment by tobiasd** [comment]
What are they for?

**Comment by annando** [comment]
@tobiasd This is part of the Mastodon API. When we had implemented all endpoints (not in this PR), then any regular Mastodon app should work with Friendica.

**Comment by tobiasd** [comment]
Could you then please update the documentation about the Mastodon API ;-)

**Comment by annando** [comment]
Meh ... I don't understand why the Transifex check fails.

**Comment by annando** [comment]
Concerning Transifex: It seems as if one of the files hadn't been updated on the repository.

**Comment by tobiasd** [comment]
Well you added some strings in the baseAPI file for translation, but you did not update the `messages.po` file.

File: src/Module/BaseApi.php [hunk 1]
```diff
@@ -66,6 +94,29 @@ public static function post(array $parameters = [])
...
+		if (!empty($a->user['uid']) && $a->user['uid'] != api_user()) {
+			throw new HTTPException\ForbiddenException(DI::l10n()->t('Permission denied.'));
+		}
+	}
+
+	public static function unsupported(string $method = 'all')
+	{
+		$path = DI::args()->getQueryString();
+		Logger::info('Unimplemented API call', ['path' => $path, 'method' => $method]);
+		$error = DI::l10n()->t('API endpoint %s "%s" is not implemented', $method, $path);
```
Comment (MrPetovan) [inline src/Module/BaseApi.php]: Please put the HTTP method in uppercase which removes the need for quotes around the path. And remember to run `bin/run_xgettext.sh` after you're done with the change. 😉
```suggestion
		$error = DI::l10n()->t('API endpoint %s %s is not implemented', strtoupper($method), $path);
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
Part of #7967

**Pull request merged**